tests/versioned/common/setup.js

```diff
@@ -17,74 +17,68 @@ const params = exports.params = {
   database: 'agent_integration_' + Math.floor(Math.random() * 1000)
 }
 
-function setup(mysql, cb) {
```

Comment:
cleaned this up a bit to just await 2 diff async functions `executeDb` and `setupDb`. My head was spinning with the nested `async.series` and `async.eachSeries`